wordpress-mobile/WordPress-Android
# Added Aztec Editor deprecation notice

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3209

### To test:
 **Create at least one Classic post using default Android Release or branch (not build from this PR) before testing**

Case 1: Existing posts
1. Navigate to existing Classic post and open it in the editor
2. Notice that there is no Classic Editor Deprecation Notice

Case 2: First time new post with Classic editor enabled
1. Check Site Settings and make sure Block Editor is disabled
2. Create New Post
3. Verify that you see the Classic Editor Deprecation Notice

Case 3: New Classic article after already seeing the dialog
1. Check Site Settings and make sure Block Editor is disabled
2. Create a new post after already having seen the dialog
3. Veryify you do NOT see the Classic Editor Deprecation Notice

### Screenshot:
<img src="https://user-images.githubusercontent.com/1103838/110185088-5223b300-7dc6-11eb-986d-57a3bb5dc597.png" alt="" width="350"> 


PR submission checklist:
